packages/monaco/src/browser/monaco-snippet-suggest-provider.ts

```diff
@@ -240,18 +244,20 @@ export interface JsonSerializedSnippets {
     [name: string]: JsonSerializedSnippet | { [name: string]: JsonSerializedSnippet };
 }
 export interface JsonSerializedSnippet {
+    isFileTemplate?: boolean;
     body: string | string[];
-    scope: string;
-    prefix: string | string[];
+    scope?: string;
+    prefix: string | string[] | undefined;
```

Comment:
Nitpick but makes the changes more consistent:

```suggestion
    scope?: string;
    prefix?: string | string[];
```

Reply:
Makes sense.